api/binaryIoTypeA/v1alpha1/binaryIoTypeA.proto

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright © 2021 Ci4Rail GmbH <engineering@ci4rail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+syntax = "proto3";
+
+package binaryIoTypeA;
+option go_package = "binaryIoTypeA/v1alpha1";
+
+// ============= ConfigurationControl =================
+// ConfigurationControlSet to pass to Functionblock.ConfigurationControl.functionSpecificConfigurationControlSet hook
+message ConfigurationControlSet {
+    // Map to enable fritting on binary output channels. This activates a cyclically higher current at the channel outputs to prevent corrison over time
+    // LSB is Output0, 1: fritting enabled, 0: fritting disabled
+    fixed32 outputFrittingMap = 2;
```

Comment:
Add outputWatchdog field for each binary output.
If != 0, enable a watchdog in the device. If the client does not call "set" within this time, the output falls back to its default state

Reply:
added